data/maps/map_groups.json

```diff
@@ -1,41 +1,41 @@
 {
   "group_order": [
-    "gMapGroup0",
-    "gMapGroup1",
-    "gMapGroup2",
-    "gMapGroup3",
-    "gMapGroup4",
-    "gMapGroup5",
-    "gMapGroup6",
-    "gMapGroup7",
-    "gMapGroup8",
-    "gMapGroup9",
-    "gMapGroup10",
-    "gMapGroup11",
-    "gMapGroup12",
-    "gMapGroup13",
-    "gMapGroup14",
-    "gMapGroup15",
-    "gMapGroup16",
-    "gMapGroup17",
-    "gMapGroup18",
-    "gMapGroup19",
-    "gMapGroup20",
-    "gMapGroup21",
-    "gMapGroup22",
-    "gMapGroup23",
-    "gMapGroup24",
-    "gMapGroup25",
-    "gMapGroup26",
-    "gMapGroup27",
-    "gMapGroup28",
-    "gMapGroup29",
-    "gMapGroup30",
-    "gMapGroup31",
-    "gMapGroup32",
-    "gMapGroup33"
-  ],
-  "gMapGroup0": [
+    "gMapGroup_TownsAndRoutes",
+    "gMapGroup_IndoorLittleroot",
```

Comment:
`Indoors` instead of `Indoor` sounds better in my head

Reply:
Seems we went with `Indoor` for the other repos